dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -884,7 +890,7 @@ export const Explorer = ({
     <div className="dscAppContainer">
       <Search
         key="search-component"
-        query={query[RAW_QUERY]}
+        query={tabId === 'application-analytics-tab' ? baseQuery : query[RAW_QUERY]}
```

Comment:
Minor: maybe use a constant for application-analytics-tab? I saw there's another line reference the same tab name couple of lines after.

Reply:
Yea that make sense!